src/Workspaces/Core/Portable/NamingStyles/EditorConfig/EditorConfigNamingStyleParser_SymbolSpec.cs

```diff
@@ -281,5 +282,206 @@ private static ImmutableArray<ModifierKind> ParseModifiers(string symbolSpecRequ
+            }
+
+            throw ExceptionUtilities.UnexpectedValue(symbol);
+        }
+
+        public static string ToEditorConfigString(this ImmutableArray<Accessibility> accessibilities, string languageName)
+        {
+            if (accessibilities.IsDefaultOrEmpty)
+            {
+                return "";
+            }
+
+            if (_allAccessibility.All(accessibilities.Contains) && accessibilities.All(_allAccessibility.Contains))
+            {
+                return "*";
+            }
+
+            return string.Join(", ", accessibilities.Select(accessibility => accessibility.ToEditorConfigString(languageName)));
+        }
+
+        private static string ToEditorConfigString(this Accessibility accessibility, string languageName)
+        {
+            switch (accessibility)
+            {
+                case Accessibility.NotApplicable:
+                    return "local";
+
+                case Accessibility.Private:
+                    return "private";
+
+                case Accessibility.ProtectedAndInternal:
+                    return "private_protected";
+
+                case Accessibility.Protected:
+                    return "protected";
+
+                case Accessibility.Internal:
+                    if (languageName == LanguageNames.VisualBasic)
+                    {
+                        return "friend";
+                    }
+                    else
+                    {
+                        return "internal";
+                    }
+
+                case Accessibility.ProtectedOrInternal:
+                    if (languageName == LanguageNames.VisualBasic)
+                    {
+                        return "protected_friend";
+                    }
+                    else
+                    {
+                        return "protected_internal";
+                    }
+
+                case Accessibility.Public:
+                    return "public";
+
+                default:
+                    throw ExceptionUtilities.UnexpectedValue(accessibility);
+            }
+        }
+
+        public static string ToEditorConfigString(this ImmutableArray<ModifierKind> modifiers, string languageName)
+        {
+            if (modifiers.IsDefaultOrEmpty)
+            {
+                return "";
+            }
+
+            if (_allModifierKind.All(modifiers.Contains) && modifiers.All(_allModifierKind.Contains))
+            {
+                return "*";
+            }
+
+            return string.Join(", ", modifiers.Select(modifier => modifier.ToEditorConfigString(languageName)));
+        }
+
+        private static string ToEditorConfigString(this ModifierKind modifier, string languageName)
+        {
+            switch (modifier.ModifierKindWrapper)
+            {
+                case ModifierKindEnum.IsAbstract:
+                    if (languageName == LanguageNames.VisualBasic)
+                    {
+                        return "must_inherit";
```

Comment:
just to check, but thsi _ was intentional?  even though it's "MustInherit" in VB as the actual token?

Reply:
➡️ This was already defined

https://github.com/dotnet/roslyn/blob/7f809efa06ad3367dbac5971c9026433be6369b6/src/Workspaces/Core/Portable/NamingStyles/EditorConfig/EditorConfigNamingStyleParser_SymbolSpec.cs#L261